source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -157,6 +158,9 @@ - (void)update:(NSString *) jsonStr
     
     if(renderResult.succeeded)
     {
+        ACRRegistration *registration = [ACRRegistration getInstance];
+        // enum will be part of API in next iterations when custom renderer extended to non-action type
```

Comment:
>enum will be part of API in next iterations when custom renderer extended to non-action type [](start = 11, length = 92)

It would be good to include an issue # here to track the work to be done. TODOs like this can easily get old and stale if we don't keep track of them.